nightingale_utils/src/room_pose_lookup.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+
+import rospy
+
+from geometry_msgs.msg import Pose
+from nightingale_msgs.srv import RoomPoseLookup, RoomPoseLookupResponse
+
+
+class RoomPoseServiceNode:
+    def __init__(self):
+        rospy.init_node("room_pose_service_node")
+
+        self.rooms = self.load_rooms()
+
+        self.server = rospy.Service(
+            "room_pose_lookup", RoomPoseLookup, self.room_pose_lookup
+        )
+
+    def load_rooms(self):
+        rooms = {}
+
+        rooms_params = rospy.get_param("rooms")
+
+        for room in rooms_params.keys():
+            features = {}
```

Comment:
Why do you use the term "features" here? Is this in preparation of some other custom traversal behaviour for each patient? I thought that we would only need the position and orientation of the robot goals since that is what we need to plan a path to the room.

Reply:
each room has an associated feature. a feature is one of {door,bedside}. this allows us to first navigate to right infront of the door, (possibly do other work like opening a door), and then going through the door and to the bedside.

This makes adding code for opening doors easier to integrate